### ISSUE TITLE
Tag index render an error url

### ISSUE THREAD

**Issue by Albert-cord** [squidfunk/mkdocs-material#3841]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [ ] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Tag index render an error url
![image](https://user-images.githubusercontent.com/31922224/165015743-d7902976-14cd-452e-9a13-7c25604d53a8.png)

### Expected behaviour

Tag index render a correct url
![image](https://user-images.githubusercontent.com/31922224/165015635-bec3cf93-9af8-4c8d-8efb-edeba5f74f32.png)

### Actual behaviour

Tag index render an error url

### Steps to reproduce

1.  code in mkdocs.yml: 
  plugins:
      - tags:
          tags_file: ./tips/tag/标签.md
  nav:
      - 菜谱:
      - 水产:
          - 白灼虾: ./dishes/aquatic/白灼虾/白灼虾.md

### Package versions

- Python: `python --version`  Python 3.10.4
- MkDocs: `mkdocs --version` 1.3.0
- Material: `pip show mkdocs-material | grep -E ^Version` Version: 8.2.10

### Configuration

```yaml
site_name: How To Cook

# Repository
repo_name: Anduin2017/HowToCook
repo_url: https://github.com/Anduin2017/HowToCook
edit_uri: ""

use_directory_urls: true
docs_dir: .
theme:
  name: material
  language: zh
  features:
    - content.code.annotate
    # - content.tabs.link
    # - header.autohide
    #- navigation.expand
    #- navigation.indexes
    - navigation.instant
    - navigation.sections
    - navigation.tabs
    - navigation.tabs.sticky
    - navigation.top
    - navigation.tracking
    - search.highlight
    - search.share
    - search.suggest
    - toc.follow
    # # - toc.integrate
  search_index_only: true
  palette:
    - media: "(prefers-color-scheme: light)"
      scheme: default
      toggle:
        icon: material//weather-sunny
        name: Switch to dark mode
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      toggle:
        icon: material/weather-night
        name: Switch to light mode

  icon:
    admonition:
      note: octicons/tag-16
      abstract: octicons/checklist-16
      info: octicons/info-16
      tip: octicons/squirrel-16
      success: octicons/check-16
      question: octicons/question-16
      warning: octicons/alert-16
      failure: octicons/x-circle-16
      danger: octicons/zap-16
      bug: octicons/bug-16
      example: octicons/beaker-16
      quote: octicons/quote-16

markdown_extensions:
  - admonition
  - pymdownx.details
  - pymdownx.superfences
  - abbr
  - pymdownx.snippets
  - def_list
  - pymdownx.tasklist:
      custom_checkbox: true
  - attr_list

plugins:
  - tags:
      tags_file: ./tips/tag/标签.md
  - same-dir
  - search
  - minify:
      minify_html: true

nav:
  - README.md
  - 做菜之前:
      - 厨房准备: ./tips/厨房准备.md
      - 如何选择现在吃什么: ./tips/如何选择现在吃什么.md
      - 上海买菜指南: ./tips/上海买菜指南.md
      - 高压力锅: ./tips/learn/高压力锅.md
      - 食品安全: ./tips/learn/食品安全.md
      - 微波炉: ./tips/learn/微波炉.md
      - 学习焯水: ./tips/learn/学习焯水.md
      - 学习炒与煎: ./tips/learn/学习炒与煎.md
      - 学习凉拌: ./tips/learn/学习凉拌.md
      - 学习腌: ./tips/learn/学习腌.md
      - 学习蒸: ./tips/learn/学习蒸.md
      - 学习煮: ./tips/learn/学习煮.md

  - ./tips/tag/标签.md

  - 菜谱:
    - 按种类: # 只有两层section以上才能出现navigation expansion https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections

      - 素菜:
          - 拔丝土豆: ./dishes/vegetable_dish/拔丝土豆/拔丝土豆.md
          - 包菜炒鸡蛋粉丝: ./dishes/vegetable_dish/包菜炒鸡蛋粉丝/包菜炒鸡蛋粉丝.md
          - 菠菜炒鸡蛋: ./dishes/vegetable_dish/菠菜炒鸡蛋/菠菜炒鸡蛋.md
          - 炒滑蛋: ./dishes/vegetable_dish/炒滑蛋/炒滑蛋.md
          - 炒茄子: ./dishes/vegetable_dish/炒茄子.md
          - 炒青菜: ./dishes/vegetable_dish/炒青菜.md
          - 葱煎豆腐: ./dishes/vegetable_dish/葱煎豆腐.md
          - 地三鲜: ./dishes/vegetable_dish/地三鲜.md
          - 干锅花菜: ./dishes/vegetable_dish/干锅花菜/干锅花菜.md
          - 蚝油三鲜菇: ./dishes/vegetable_dish/蚝油三鲜菇/蚝油三鲜菇.md
          - 蚝油生菜: ./dishes/vegetable_dish/蚝油生菜.md
          - 红烧冬瓜: ./dishes/vegetable_dish/红烧冬瓜/红烧冬瓜.md
          - 红烧茄子: ./dishes/vegetable_dish/红烧茄子.md
          - 虎皮青椒: ./dishes/vegetable_dish/虎皮青椒/虎皮青椒.md
          - 话梅煮毛豆: ./dishes/vegetable_dish/话梅煮毛豆/话梅煮毛豆.md
          - 鸡蛋羹: ./dishes/vegetable_dish/鸡蛋羹/鸡蛋羹.md
          - 微波炉鸡蛋羹: ./dishes/vegetable_dish/鸡蛋羹/微波炉鸡蛋羹.md
          - 鸡蛋火腿炒黄瓜: ./dishes/vegetable_dish/鸡蛋火腿炒黄瓜.md
          - 茄子炖土豆: ./dishes/vegetable_dish/茄子炖土豆.md
          - 酱拌荞麦面: ./dishes/vegetable_dish/酱拌荞麦面/酱拌荞麦面.md
          - 茭白炒肉: ./dishes/vegetable_dish/茭白炒肉/茭白炒肉.md
          - 椒盐玉米: ./dishes/vegetable_dish/椒盐玉米/椒盐玉米.md
          - 烤茄子: ./dishes/vegetable_dish/烤茄子/烤茄子.md
          - 榄菜肉末四季豆: ./dishes/vegetable_dish/榄菜肉末四季豆/榄菜肉末四季豆.md
          - 凉拌黄瓜: ./dishes/vegetable_dish/凉拌黄瓜.md
          - 凉拌莴笋: ./dishes/vegetable_dish/凉拌莴笋/凉拌莴笋.md
          - 凉拌油麦菜: ./dishes/vegetable_dish/凉拌油麦菜.md
          - 麻婆豆腐: ./dishes/vegetable_dish/麻婆豆腐/麻婆豆腐.md
          - 蒲烧茄子: ./dishes/vegetable_dish/蒲烧茄子.md
          - 陕北熬豆角: ./dishes/vegetable_dish/陕北熬豆角.md
          - 上汤娃娃菜: ./dishes/vegetable_dish/上汤娃娃菜/上汤娃娃菜.md
          - 手撕包菜: ./dishes/vegetable_dish/手撕包菜/手撕包菜.md
          - 水油焖蔬菜: ./dishes/vegetable_dish/水油焖蔬菜.md
          - 酸辣土豆丝: ./dishes/vegetable_dish/酸辣土豆丝.md
          - 糖拌西红柿: ./dishes/vegetable_dish/糖拌西红柿/糖拌西红柿.md
          - 西红柿炒鸡蛋: ./dishes/vegetable_dish/西红柿炒鸡蛋.md
          - 西葫芦炒鸡蛋: ./dishes/vegetable_dish/西葫芦炒鸡蛋/西葫芦炒鸡蛋.md
          - 洋葱炒鸡蛋: ./dishes/vegetable_dish/洋葱炒鸡蛋/洋葱炒鸡蛋.md

      - 荤菜:
          - 白菜猪肉炖粉条: ./dishes/meat_dish/白菜猪肉炖粉条.md
          - 番茄红酱: ./dishes/meat_dish/番茄红酱.md
          - 干煸仔鸡: ./dishes/meat_dish/干煸仔鸡/干煸仔鸡.md
          - 宫保鸡丁: ./dishes/meat_dish/宫保鸡丁/宫保鸡丁.md
          - 咕噜肉: ./dishes/meat_dish/咕噜肉.md
          - 黑椒牛柳: ./dishes/meat_dish/黑椒牛柳/黑椒牛柳.md
          - 简易红烧肉: ./dishes/meat_dish/红烧肉/简易红烧肉.md
          - 南派红烧肉: ./dishes/meat_dish/红烧肉/南派红烧肉.md
          - 红烧猪蹄: ./dishes/meat_dish/红烧猪蹄/红烧猪蹄.md
          - 湖南家常红烧肉: ./dishes/meat_dish/湖南家常红烧肉/湖南家常红烧肉.md
          - 黄瓜炒肉: ./dishes/meat_dish/黄瓜炒肉.md
          - 黄焖鸡: ./dishes/meat_dish/黄焖鸡.md
          - 徽派红烧肉: ./dishes/meat_dish/徽派红烧肉/徽派红烧肉.md
          - 回锅肉: ./dishes/meat_dish/回锅肉/回锅肉.md
          - 尖椒炒牛肉: ./dishes/meat_dish/尖椒炒牛肉.md
          - 姜炒鸡: ./dishes/meat_dish/姜炒鸡/姜炒鸡.md
          - 酱牛肉: ./dishes/meat_dish/酱牛肉/酱牛肉.md
          - 酱排骨: ./dishes/meat_dish/酱排骨/酱排骨.md
          - 可乐鸡翅: ./dishes/meat_dish/可乐鸡翅.md
          - 口水鸡: ./dishes/meat_dish/口水鸡/口水鸡.md
          - 辣椒炒肉: ./dishes/meat_dish/辣椒炒肉.md
          - 老式锅包肉: ./dishes/meat_dish/老式锅包肉/老式锅包肉.md
          - 冷吃兔: ./dishes/meat_dish/冷吃兔.md
          - 荔枝肉: ./dishes/meat_dish/荔枝肉/荔枝肉.md
          - 凉拌鸡丝: ./dishes/meat_dish/凉拌鸡丝/凉拌鸡丝.md
          - 萝卜炖羊排: ./dishes/meat_dish/萝卜炖羊排.md
          - 麻辣香锅: ./dishes/meat_dish/麻辣香锅.md
          - 梅菜扣肉: ./dishes/meat_dish/梅菜扣肉/梅菜扣肉.md
          - 啤酒鸭: ./dishes/meat_dish/啤酒鸭/啤酒鸭.md
          - 水煮牛肉: ./dishes/meat_dish/水煮牛肉/水煮牛肉.md
          - 水煮肉片: ./dishes/meat_dish/水煮肉片.md
          - 蒜苔炒肉末: ./dishes/meat_dish/蒜苔炒肉末.md
          - 糖醋里脊: ./dishes/meat_dish/糖醋里脊.md
          - 糖醋排骨: ./dishes/meat_dish/糖醋排骨/糖醋排骨.md
          - 土豆炖排骨: ./dishes/meat_dish/土豆炖排骨/土豆炖排骨.md
          - 无骨鸡爪: ./dishes/meat_dish/无骨鸡爪/无骨鸡爪.md
          - 西红柿牛腩: ./dishes/meat_dish/西红柿牛腩/西红柿牛腩.md
          - 西红柿土豆炖牛肉: ./dishes/meat_dish/西红柿土豆炖牛肉/西红柿土豆炖牛肉.md
          - 香干芹菜炒肉: ./dishes/meat_dish/香干芹菜炒肉/香干芹菜炒肉.md
          - 香干肉丝: ./dishes/meat_dish/香干肉丝.md
          - 香菇滑鸡: ./dishes/meat_dish/香菇滑鸡/香菇滑鸡.md
          - 小炒黄牛肉: ./dishes/meat_dish/小炒黄牛肉/小炒黄牛肉.md
          - 小炒肉: ./dishes/meat_dish/小炒肉.md
          - 新疆大盘鸡: ./dishes/meat_dish/新疆大盘鸡/新疆大盘鸡.md
          - 血浆鸭: ./dishes/meat_dish/血浆鸭/血浆鸭.md
          - 羊排焖面: ./dishes/meat_dish/羊排焖面/羊排焖面.md
          - 洋葱炒猪肉: ./dishes/meat_dish/洋葱炒猪肉.md
          - 鱼香茄子: ./dishes/meat_dish/鱼香茄子/鱼香茄子.md
          - 鱼香肉丝: ./dishes/meat_dish/鱼香肉丝.md
          - 猪皮冻: ./dishes/meat_dish/猪皮冻/猪皮冻.md
          - 孜然牛肉: ./dishes/meat_dish/孜然牛肉.md
          - 醉排骨: ./dishes/meat_dish/醉排骨/醉排骨.md

      - 水产:
          - 白灼虾: ./dishes/aquatic/白灼虾/白灼虾.md
          - 鳊鱼炖豆腐: ./dishes/aquatic/鳊鱼炖豆腐/鳊鱼炖豆腐.md
          - 蛏抱蛋: ./dishes/aquatic/蛏抱蛋/蛏抱蛋.md
          - 葱烧海参: ./dishes/aquatic/葱烧海参/葱烧海参.md
          - 红烧鲤鱼: ./dishes/aquatic/红烧鲤鱼.md
          - 红烧鱼头: ./dishes/aquatic/红烧鱼头.md
          - 黄油煎虾: ./dishes/aquatic/黄油煎虾/黄油煎虾.md
          - 烤鱼: ./dishes/aquatic/混合烤鱼/烤鱼.md
          - 咖喱炒蟹: ./dishes/aquatic/咖喱炒蟹.md
          - 清蒸鲈鱼: ./dishes/aquatic/清蒸鲈鱼/清蒸鲈鱼.md
          - 清蒸生蚝: ./dishes/aquatic/清蒸生蚝.md
          - 蒜蓉虾: ./dishes/aquatic/蒜蓉虾/蒜蓉虾.md
          - 糖醋鲤鱼: ./dishes/aquatic/糖醋鲤鱼/糖醋鲤鱼.md
          - 微波葱姜黑鳕鱼: ./dishes/aquatic/微波葱姜黑鳕鱼.md
          - 香煎翘嘴鱼: ./dishes/aquatic/香煎翘嘴鱼/香煎翘嘴鱼.md
          - 小龙虾: ./dishes/aquatic/小龙虾/小龙虾.md

      - 早餐:
          - 茶叶蛋: ./dishes/breakfast/茶叶蛋.md
          - 桂圆红枣粥: ./dishes/breakfast/桂圆红枣粥.md
          - 鸡蛋三明治: ./dishes/breakfast/鸡蛋三明治.md
          - 煎饺: ./dishes/breakfast/煎饺.md
          - 空气炸锅面包片: ./dishes/breakfast/空气炸锅面包片.md
          - 美式炒蛋: ./dishes/breakfast/美式炒蛋.md
          - 牛奶燕麦: ./dishes/breakfast/牛奶燕麦.md
          - 水煮玉米: ./dishes/breakfast/水煮玉米.md
          - 苏格兰蛋: ./dishes/breakfast/苏格兰蛋/苏格兰蛋.md
          - 太阳蛋: ./dishes/breakfast/太阳蛋.md
          - 溏心蛋: ./dishes/breakfast/溏心蛋.md
          - 吐司果酱: ./dishes/breakfast/吐司果酱.md
          - 微波炉蛋糕: ./dishes/breakfast/微波炉蛋糕.md
          - 燕麦鸡蛋饼: ./dishes/breakfast/燕麦鸡蛋饼.md
          - 蒸花卷: ./dishes/breakfast/蒸花卷.md
          - 蒸水蛋: ./dishes/breakfast/蒸水蛋.md

      - 主食:
          - 炒方便面: ./dishes/staple/炒方便面.md
          - 炒河粉: ./dishes/staple/炒河粉.md
          - 炒馍: ./dishes/staple/炒馍.md
          - 炒年糕: ./dishes/staple/炒年糕.md
          - 炒意大利面: ./dishes/staple/炒意大利面/炒意大利面.md
          - 蛋炒饭: ./dishes/staple/蛋炒饭.md
          - 韩式拌饭: ./dishes/staple/韩式拌饭/韩式拌饭.md
          - 基础牛奶面包: ./dishes/staple/基础牛奶面包/基础牛奶面包.md
          - 茄子肉煎饼: ./dishes/staple/茄子肉煎饼/茄子肉煎饼.md
          - 醪糟小汤圆: ./dishes/staple/醪糟小汤圆.md
          - 老干妈拌面: ./dishes/staple/老干妈拌面.md
          - 老友猪肉粉: ./dishes/staple/老友猪肉粉/老友猪肉粉.md
          - 烙饼: ./dishes/staple/烙饼/烙饼.md
          - 凉粉: ./dishes/staple/凉粉/凉粉.md
          - 麻辣减脂荞麦面: ./dishes/staple/麻辣减脂荞麦面.md
          - 麻油拌面: ./dishes/staple/麻油拌面.md
          - 电饭煲蒸米饭: ./dishes/staple/米饭/电饭煲蒸米饭.md
          - 煮锅蒸米饭: ./dishes/staple/米饭/煮锅蒸米饭.md
          - 披萨饼皮: ./dishes/staple/披萨饼皮/披萨饼皮.md
          - 热干面: ./dishes/staple/热干面.md
          - 日式咖喱饭: ./dishes/staple/日式咖喱饭/日式咖喱饭.md
          - 烧饼: ./dishes/staple/烧饼/烧饼.md
          - 手工水饺: ./dishes/staple/手工水饺.md
          - 酸辣蕨根粉: ./dishes/staple/酸辣蕨根粉.md
          - 汤面: ./dishes/staple/汤面.md
          - 微波炉腊肠煲仔饭: ./dishes/staple/微波炉腊肠煲仔饭/微波炉腊肠煲仔饭.md
          - 炸酱面: ./dishes/staple/炸酱面.md
          - 蒸卤面: ./dishes/staple/蒸卤面.md
          - 中式馅饼: ./dishes/staple/中式馅饼/中式馅饼.md
          - 煮泡面加蛋: ./dishes/staple/煮泡面加蛋.md

      - 半成品加工:
          - 半成品意面: ./dishes/semi-finished/半成品意面.md
          - 懒人蛋挞: ./dishes/semi-finished/懒人蛋挞/懒人蛋挞.md
          - 凉皮: ./dishes/semi-finished/凉皮.md
          - 牛油火锅底料: ./dishes/semi-finished/牛油火锅底料.md
          - 速冻馄饨: ./dishes/semi-finished/速冻馄饨.md
          - 速冻水饺: ./dishes/semi-finished/速冻水饺.md
          - 速冻汤圆: ./dishes/semi-finished/速冻汤圆/速冻汤圆.md
          - 炸薯条: ./dishes/semi-finished/炸薯条/炸薯条.md

      - 汤与粥:
          - 昂刺鱼豆腐汤: ./dishes/soup/昂刺鱼豆腐汤/昂刺鱼豆腐汤.md
          - 勾芡香菇汤: ./dishes/soup/勾芡香菇汤/勾芡香菇汤.md
          - 金针菇汤: ./dishes/soup/金针菇汤.md
          - 菌菇炖乳鸽: ./dishes/soup/菌菇炖乳鸽/菌菇炖乳鸽.md
          - 罗宋汤: ./dishes/soup/罗宋汤.md
          - 米粥: ./dishes/soup/米粥.md
          - 皮蛋瘦肉粥: ./dishes/soup/皮蛋瘦肉粥.md
          - 生汆丸子汤: ./dishes/soup/生汆丸子汤.md
          - 西红柿鸡蛋汤: ./dishes/soup/西红柿鸡蛋汤.md
          - 小米粥: ./dishes/soup/小米粥.md
          - 银耳莲子粥: ./dishes/soup/银耳莲子粥/银耳莲子粥.md
          - 紫菜蛋花汤: ./dishes/soup/紫菜蛋花汤.md

      - 饮料:
          - 耙耙柑茶: ./dishes/drink/耙耙柑茶/耙耙柑茶.md
          - 百香果橙子特调: ./dishes/drink/百香果橙子特调/百香果橙子特调.md
          - 金菲士: ./dishes/drink/金菲士/金菲士.md
          - 金汤力: ./dishes/drink/金汤力/金汤力.md
          - 可乐桶: ./dishes/drink/可乐桶.md
          - 奶茶: ./dishes/drink/奶茶.md
          - 奇异果菠菜特调: ./dishes/drink/奇异果菠菜特调/奇异果菠菜特调.md
          - 酸梅汤: ./dishes/drink/酸梅汤/酸梅汤.md
          - 酸梅汤（半成品加工）: ./dishes/drink/酸梅汤（半成品加工）.md
          - 杨枝甘露: ./dishes/drink/杨枝甘露.md
          - 长岛冰茶: ./dishes/drink/长岛冰茶.md
          - B52轰炸机: ./dishes/drink/B52轰炸机.md
          - Mojito莫吉托: ./dishes/drink/Mojito莫吉托.md

      - 酱料和其它材料:
          - 草莓酱: ./dishes/condiment/草莓酱/草莓酱.md
          - 蒜香酱油: ./dishes/condiment/蒜香酱油.md
          - 糖醋汁: ./dishes/condiment/糖醋汁.md
          - 糖色: ./dishes/condiment/糖色.md
          - 油泼辣子: ./dishes/condiment/油泼辣子/油泼辣子.md
          - 油酥: ./dishes/condiment/油酥.md
          - 炸串酱料: ./dishes/condiment/炸串酱料.md
          - 蔗糖糖浆: ./dishes/condiment/蔗糖糖浆/蔗糖糖浆.md

      - 甜品:
          - 奥利奥冰淇淋: ./dishes/dessert/奥利奥冰淇淋/奥利奥冰淇淋.md
          - 烤蛋挞: ./dishes/dessert/烤蛋挞/烤蛋挞.md
          - 魔芋蛋糕: ./dishes/dessert/魔芋蛋糕/魔芋蛋糕.md
          - 戚风蛋糕: ./dishes/dessert/戚风蛋糕/戚风蛋糕.md
          - 提拉米苏: ./dishes/dessert/提拉米苏/提拉米苏.md
          - 雪花酥: ./dishes/dessert/雪花酥/雪花酥.md
          - 芋泥雪媚娘: ./dishes/dessert/芋泥雪媚娘/芋泥雪媚娘.md

  - 进阶知识学习:
      - 辅料技巧: ./tips/advanced/辅料技巧.md
      - 高级专业术语: ./tips/advanced/高级专业术语.md
      - 油温判断技巧: ./tips/advanced/油温判断技巧.md

  - CONTRIBUTING.md
  - CODE_OF_CONDUCT.md
```

### System information

- Operating system: Win10
- Browser:  Google Chrome 100.0.4896.127

**Comment by Albert-cord** [comment]
I think this line of code is causing the problem. 
https://github.com/squidfunk/mkdocs-material/blob/87671c94e6c15c6cc7867dd161adc13ffa0778bc/src/plugins/tags/plugin.py#L108

**Comment by squidfunk** [comment]
Thanks for reporting. Please provide a minimal full self-contained example. You can upload it as a zip file to this issue.

**Comment by Albert-cord** [comment]
[bug_report.zip](https://github.com/squidfunk/mkdocs-material/files/8556054/bug_report.zip)
This is a minimal fully self-contained example.

**Comment by squidfunk** [comment]
Thanks for providing. However, the example does not work. There seems to be some problem with the character set:

``` sh
(venv) bug_report > ls tips/tag
ąęĮŠ.md # should be 标签.md
```

If I rename the file to `标签.md`, everything works as expected. This is not an error with Material for MkDocs, but probably related to using non-UTF-8 character set when naming the file. If I rename the file, I'm greeted by the tags page:

<img width="1532" alt="Bildschirmfoto 2022-04-25 um 20 27 10" src="https://user-images.githubusercontent.com/932156/165151448-9e7387fd-4810-417f-92e4-5f838136cd4d.png">

Closing, as it works as expected.

**Comment by Albert-cord** [comment]
Thks, I know why this is causing this problem, because of this line of code below mkdoc. 
https://github.com/mkdocs/mkdocs/blob/cdf8a26cafa6af6cc78a45766dfec235bd7286cc/mkdocs/utils/__init__.py#L15
and then use it for utils.get_relative_url.
https://github.com/mkdocs/mkdocs/blob/cdf8a26cafa6af6cc78a45766dfec235bd7286cc/mkdocs/utils/__init__.py#L248
I mean all of this issue just for blaming why I use windows operation system. 
All in all, thanks.

**Comment by squidfunk** [comment]
Maybe then raise this issue over at MkDocs? We're just using what MkDocs offers.

**Comment by squidfunk** [comment]
We found and fixed the problem in #3865. Sorry that it took some time, but I was not able to debug on Windows.

**Comment by squidfunk** [comment]
Released as part of 8.2.13.
